makeflow/src/dag_node.c

```diff
@@ -341,14 +341,19 @@ struct jx * dag_node_env_create( struct dag *d, struct dag_node *n )
 	char *key;
 
 	struct jx *object = jx_object(0);
+    
+    char *num_cores = dag_variable_lookup_string(RESOURCES_CORES, &s);
```

Comment:
Generally speaking, check for null anytime you have a pointer returned from a function.
If it's null, then `CORES` isn't set.